testsuite/helpers/retcode.cc

```diff
@@ -0,0 +1,15 @@
+// Return the return code given as command line argument as exit code.
+
+#include <stdlib.h>
+
+int main ( int argc, char *argv[] )
+{
+    int retCode = 0;
+    
+    if ( argc > 1 )
+    {
+	retCode = atoi( argv[1] );
```

Comment:
That would throw an exception for illegal input which would need to be handled. Too much unnecessary complexity for a minimalistic helper like this IMHO. A value of 0 will do just nicely for illegal input here.